docs/talk2biomodels/cases/Case_1.md

```diff
@@ -0,0 +1,156 @@
+# Example Use Case #1
+## Crohn's Disease model
+
+**Paper:**
+> Dwivedi G, Fitz L, Hegen M, Martin SW, Harrold J, Heatherington A, Li C. (2014). A multiscale model of interleukin-6-mediated immune regulation in Crohn's disease and its application in drug discovery and development. *CPT Pharmacometrics Syst Pharmacol*, 3(1):e89. doi: 10.1038/psp.2013.64. [PMID: 24402116](https://pubmed.ncbi.nlm.nih.gov/24402116/)
+
+**Model**: [BIOMD0000000537](https://www.ebi.ac.uk/biomodels/BIOMD0000000537)
+
+<hr>
+
+### General questions about the model
+
+<hr>
+
+**Q1:** Search for a model on BioModels database related to a search term <br>
+
+```Search for all the BioModels on Crohn's Disease```
```

Comment:
At least for me, I find code snippets easier to read (and copy-n-paste) if they are chunked into code blocks like the following:

```
Search all the BioModels on Crohn's Disease
```